src/components/checkbox/examples/checkboxes.vue

```diff
@@ -135,7 +135,6 @@
         Select All
       </cdr-checkbox>
       <cdr-list
-        role="group"
```

Comment:
I think this should technically still be valid but Axe didn't like it so removing the group role.